### ISSUE TITLE
Add option to remove early stopping

### ISSUE THREAD

**Issue by MatthewAlmeida** [MatthewAlmeida/PACS_Sampling_Experiments#5]
I'm seeing some models (esp. those with relatively high learning rate) have a "best" model from too early in training; these seem to overfit the validation set to some degree.  Given the problem domain, there is little advantage to fitting the validation set hard.
